Dockerfile

```diff
@@ -12,6 +12,13 @@ RUN apt-get install ghostscript openjdk-8-jre-headless wget git unzip xpdf-utils
 
 WORKDIR /opt/strip-coverletter
 
+# create a worker to run the script as
+RUN useradd worker --uid 1000 --shell /bin/bash --no-create-home
+RUN chown worker .
```

Comment:
probably want to change the group too?

Reply:
not really? permissions should be `1000:0` after chown. Is there any benefit to also being group owner?

Reply:
just consistency, as it makes the directory the same as if it was created by that user